### ISSUE TITLE
Booking Overview - UI

### ISSUE THREAD

**Issue by qy21gafy** [qy21gafy/AMOSSharedDeskFeatureBoard#15]
## User story
1. As a user
2. I want / need to see my bookings
3. So that I remember when to go to the office and can delete or edit the bookings if necessary

## Acceptance criteria
* The UI page should be identifyable as the booking overview page (e.g through header)
* The UI page should display all bookings with the desk, date and time details for that user ordered by date/time
* The page should provide access to some edit and delete functionality for the bookings

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
